examples/with-remix-thirdpartyemailpassword/README.md

```diff
@@ -0,0 +1,4 @@
+```shell
```

Comment:
need to add a better readme. It needs to communicate the features the example app has and also the framework being used.

Reply:
Done.